Orange/datasets/zoo-with-images.tab

```diff
@@ -0,0 +1 @@
+catsize	predator	aquatic	fins	feathers	type1	name	images
d	d	d	d	d	d	string	string
					class	meta	meta type=image
1	0	0	0	0	mammal	antelope	http://icons.iconarchive.com/icons/joseph-aeron/us-fish-and-wildlife-service/128/antelope-icon.png
0	1	1	1	0	fish	bass	http://images2.fanpop.com/images/photos/5700000/Largemouth-Bass-fishing-5708828-120-106.jpg
1	1	0	0	0	mammal	bear	http://icons.iconarchive.com/icons/iconshock/alaska/256/Polar-Bear-icon.png
1	1	0	0	0	mammal	boar	https://pbs.twimg.com/profile_images/2967299392/e0aa28ab427452deacc567a8f2816b3f.jpeg
0	0	1	1	0	fish	carp	http://www.landbigfish.com/images/fish/LBF_Common_Carp.jpg
0	1	1	1	0	fish	catfish	http://www.agfc.com/speciesPhotos/fish_catfish_yellowbullhead.jpg
0	0	0	0	1	bird	chicken	http://latimesblogs.latimes.com/.a/6a00d8341c630a53ef01156f75357a970c-400wi
1	0	0	0	0	mammal	deer	http://icons.iconarchive.com/icons/joseph-aeron/us-fish-and-wildlife-service/128/deer-icon.png
1	1	1	1	0	mammal	dolphin	http://f0.pepst.com/c/D7D076/36618/ssc3/home/023/deepakjain/albums/dolphin_14kb.jpg_480_480_0_64000_0_1_0.jpg
0	0	1	0	1	bird	duck	http://i.dailymail.co.uk/i/pix/2009/05/21/article-1185197-05075EEF000005DC-380_468x286.jpg
0	1	1	0	1	bird	gull	http://www.allaboutbirds.org/guide/PHOTO/LARGE/herring_gull_adult_breeding2.jpg
0	0	1	1	0	fish	haddock	http://www.oceantrawlers.com/sites/default/files/haddock_0.png?1317994124
0	0	0	0	0	mammal	hamster	http://img3.wikia.nocookie.net/__cb20130325185045/animalcrossing/images/4/49/Tumblr_lvrcmvCpsS1qbeyouo1_500.jpg
0	1	0	0	1	bird	kiwi	http://i.telegraph.co.uk/multimedia/archive/01891/kiwi_1891642c.jpg
1	1	1	0	0	mammal	mink	http://i.dailymail.co.uk/i/pix/2012/05/15/article-2144681-0D4F15C800000578-601_468x370.jpg

```

Comment:
Would it make sense to host the images in our own imgur (or similar) album? That way, the images are future-safe, or at least we get a notice, when imgur eventually shuts down.